packages/manager/modules/pci/src/components/index.js

```diff
@@ -1,11 +1,21 @@
 import angular from 'angular';
 
+import cucConsumption from './consumption';
+import cucCurrency from './currency';
+import cucFlavor from './flavor';
 import project from './project';
 import runabove from './runabove';
 import writeRightRequired from './writeRightRequired';
 
 const moduleName = 'ovhManagerPciComponents';
 
-angular.module(moduleName, [project, runabove, writeRightRequired]);
+angular.module(moduleName, [
+  cucConsumption,
+  cucCurrency,
+  cucFlavor,
```

Comment:
If I have a look this is only used for one function of one service across all manager, maybe we can just replace its usage

Reply:
i just deleted cucFlavor module and copied the "addOverQuotasInfos" where it's used (in kubernetes node-pool nodes add)